validator/client/validator_test.go

```diff
@@ -15,7 +19,35 @@ var _ = Validator(&validator{})
 
 var fakePubKey = []byte{1}
 
-func TestUpdateAssignmentsDoesNothingWhenNotEpochStart(t *testing.T) {
+func TestWaitForChainStart_ReceivesChainStartGenesisTime(t *testing.T) {
```

Comment:
What behavior are you testing? That the chain start receives a genesis time from the server? That is mocked behavior, please consider a more meaningful name i.e. `..._SetsGenesisTime`